docs/articles/vs-test-adapter/Trace-and-Debug.md

```diff
@@ -0,0 +1,66 @@
+# How to enable Trace and Debug output
+
+Trace and Debug output is by default not sent to the console output, or anywhere else.  If you want to have this output, you must add your own tracelistener.
+
+## Implementing the tracelistener in your code
+
+This is easy to do, and you can do it per assembly, per namespace or per test.  All it takes is to add the necessary initialization of the tracelistener.
+
+To add it to a namespace, place the following class inside that namespace, and it will apply to all tests inside that namespace and below.  The code shown adds the output to console output.  If you like you can change that to another kind of listener.
+
+```cs
+[SetUpFixture]
+public class SetupTrace
+{
+    [OneTimeSetUp]
+    public void StartTest()
+    {
+        Trace.Listeners.Add(new ConsoleTraceListener());
+    }
+
+    [OneTimeTearDown]
+    public void EndTest()
+    {
+        Trace.Flush();
+    }
+}
+```
+
+If you place the code above `outside any namespace` it will apply to the whole assembly.
```

Comment:
```suggestion
If you place the code above outside of any namespace it will apply to the whole assembly.
```

Removing the backticks here as it does not refer to code or an explicit choice as far as I can tell.